packages/terra-framework-docs/src/terra-dev-site/doc/pills/AccessibilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,99 @@
+import { Badge } from 'terra-pills/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
+
+<Badge />
+
+# Accessibility Guide for Terra Filter Pills
+
+Filter Pills have special interactions and construction that makes them unique from standard buttons and interactive list items.
+
+## Important Considerations:
+
+### Construction of Group Label / Heading
+
+- Filter Pills as a group requires that an `aria-label` be provided as accessibility minimum standard protection, but is not visible, and therefore is not reccomended to be used alone.
+- For accessibility best practices, it is recommended that consumers should always include a visible label by using the `ariaLabelledBy` prop.
+- Filter Pills are commonly used in a group or list, which must include a label to indicate the intent or purpose of the contents of the list. Accordingly, the filter pills component requires the `ariaLabel` prop for accessibility minimum standards, to give a screenreader user the proper context of the list, but it is not visible to users.
+
+
+<Notice variant="important" ariaLevel="1">
+Accessibility Considerations:
+
+For best practice, it is recommended that pills include a visible label, which may need to be styled and positioned according to a specific usage and should be done so by each consumer. Use the `ariaLabelledBy` prop in Filter Pills, and provide it the ʻidʼ of the visible label or heading, which will allow for the visible text to be accessibly connected to the pills.
```

Comment:
```suggestion
For best practice, it is recommended that pills include a visible label, which may need to be styled and positioned  by the consumer according to their specific usage. Use the `ariaLabelledBy` prop in Filter Pills and provide it the ʻidʼ of the visible label or heading. This will allow for the visible text to be accessibly connected to the pills.
```

Reply:
[updated](https://github.com/cerner/terra-framework/pull/1490/commits/8d775bb7b611089a8d939625bd2832781544462a#diff-c0f356f3f58def9dbe21de651bee38f73c27a6b729e3d1170e20542793c04538)